src/main/java/com/es/phoneshop/validator/OrderPhoneValidator.java

```diff
@@ -0,0 +1,23 @@
+package com.es.phoneshop.validator;
+
+public class OrderPhoneValidator {
```

Comment:
we need at least one more validator for dates
now it's possible to set delivery date in the past